apps/src/code-studio/components/progress/progress_dot.jsx

```diff
@@ -198,10 +198,11 @@ export const ProgressDot = Radium(React.createClass({
     const showUnplugged = isUnplugged && (this.props.courseOverviewPage || onCurrent);
     const outlineCurrent = this.props.courseOverviewPage && onCurrent;
     const smallDot = !this.props.courseOverviewPage && !onCurrent;
-    const showLevelName = /(named_level|peer_review)/.test(level.kind) && this.props.courseOverviewPage;
-    const isPeerReview = level.kind === 'peer_review';
+    const showLevelName = this.props.courseOverviewPage &&
+      (level.kind === LevelKind.named_level || level.kind === LevelKind.peer_review);
+    const isPeerReview = level.kind === LevelKind.peer_review;
     // Account for both the level based concept of locked, and the progress based concept.
-    const isLocked = level.locked || levelStatus === LevelStatus.locked;
+    const isLocked = levelStatus === LevelStatus.locked;
```

Comment:
One benefit of this change is we no longer need to look at level.locked, since levelStatus will now be `LevelStatus.locked` even for peer reviews (something that wasn't true before).